src/Native/Bootstrap/main.cpp

```diff
@@ -293,15 +293,58 @@ static const pfn c_classlibFunctions[] = {
 #endif // !CPPCODEGEN
 
 extern "C" void InitializeModules(void* osModule, void ** modules, int count, void ** pClasslibFunctions, int nClasslibFunctions);
+extern "C" int InitializeRuntime();
```

Comment:
You can move the implementation of InitializeRuntime up to avoid the forward declaration.